pkg/sqlstore/migrations/007_add_integrations.go

```diff
@@ -0,0 +1,59 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/migrate"
+	"go.signoz.io/signoz/pkg/factory"
+	"go.signoz.io/signoz/pkg/sqlstore"
+)
+
+type addIntegrations struct {
+	settings factory.ProviderSettings
+}
+
+func NewAddIntegrationsFactory() factory.ProviderFactory[sqlstore.Migration, sqlstore.Config] {
+	return factory.NewProviderFactory(factory.MustNewName("add_integrations"), newAddIntegrations)
+}
+
+func newAddIntegrations(_ context.Context, settings factory.ProviderSettings, _ sqlstore.Config) (sqlstore.Migration, error) {
+	return &addIntegrations{settings: settings}, nil
+}
+
+func (migration *addIntegrations) Register(migrations *migrate.Migrations) error {
+	if err := migrations.Register(migration.Up, migration.Down); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (migration *addIntegrations) Up(ctx context.Context, db *bun.DB) error {
+	if _, err := db.ExecContext(ctx, `CREATE TABLE IF NOT EXISTS integrations_installed(
+		integration_id TEXT PRIMARY KEY,
+		config_json TEXT,
+		installed_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+	);`); err != nil {
+		return err
+	}
+
+	if _, err := db.ExecContext(ctx, `CREATE TABLE IF NOT EXISTS cloud_integrations_accounts(
```

Comment:
since these are already known things, I guess we can club this all to the `001_migration_squashed`, anything new can be added as a new migration.